File: WebApp/Scripts/app/model/AA/AA0184.js
Ext.define('WEBAPP.model.AA.AA0184', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'DOCNO', type: 'string' },//申請單號
        { name: 'MAT_CLASS', type: 'string' },//物料分類
        { name: 'COMBITEM', type: 'string' },//狀態
        { name: 'FRWH', type: 'string' },//出庫庫房
        { name: 'TOWH', type: 'string' },//入庫庫房
        { name: 'APPTIME', type: 'string' },//申請時間
        { name: 'MMCODE', type: 'string' },//院內碼
        { name: 'MMNAME_C', type: 'string' },//中文名稱
        { name: 'MMNAME_E', type: 'string' },//英文名稱
        { name: 'BASE_UNIT', type: 'string' },//單位
        { name: 'APPQTY', type: 'string' },//申請數量
        { name: 'APVQTY', type: 'string' },//核可數量
        { name: 'DIS_TIME', type: 'string' }//核撥日期
    ]
});

File: WebApp/Models/AA/AA0184.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models.AB
{
    public class AA0184 : JCLib.Mvc.BaseModel
    {
        public string DOCNO { get; set; }//申請單號
        public string MAT_CLASS { get; set; }//物料分類
        public string COMBITEM { get; set; }//狀態
        public string FRWH { get; set; }//出庫庫房
        public string TOWH { get; set; }//入庫庫房
        public string APPTIME { get; set; }//申請時間
        public string MMCODE { get; set; }//院內碼
        public string MMNAME_C { get; set; }//中文名稱
        public string MMNAME_E { get; set; }//英文名稱
        public string BASE_UNIT { get; set; }//單位
        public string APPQTY { get; set; }//申請數量
        public string APVQTY { get; set; }//核可數量
        public string DIS_TIME { get; set; }//核撥日期
    }
}
